.travis.yml

```diff
@@ -1,21 +1,30 @@
-sudo: false
 language: python
 dist: xenial
-python:
-  - 2.7
-  - 3.5
-  - 3.6
-  - 3.7
-  - 3.8-dev
+stages:
+    - linting
```

Comment:
separate stages can be useful in some cases, but may frustrate your contributors (_personally_ I think things should be lint-clean before testing, but some like to throw code at CI just to see what sticks 🤷‍♂)

Reply:
Interesting. So you are saying that it may frustrate people if their code passes all the tests but the linting does not allow them to move to the next stage of CI?

Reply:
nah, more that some times people want to throw a change at CI to see whether it passes tests or not, and not care about style until later -- if you have a blocking lint stage it would prevent that use case

Reply:
Fair point. I have removed stages and went back to simply naming the linting stage with `name: "Linting"`.